content/posts.md

```diff
@@ -0,0 +1,7 @@
++++
+title = 'Posts'
+date = 2023-10-21T22:18:56+05:30
+draft = true
```

Comment:
Have marked this as draft, so the website won't be visible. Instead throws a 404